01_draw_functions/01_subfunctions/SensorPlots.m

```diff
@@ -33,7 +33,7 @@ function SensorPlots(sysvector, topics, fconv_gpsalt)
     legend('x','y','z');
     raw(4)=subplot(4,1,4);
     hold on;
-    plot(sysvector('sensor_combined_0.baro_alt_meter').Time,sysvector('sensor_combined_0.baro_alt_meter').Data);
+    plot(sysvector('vehicle_air_data.baro_alt_meter').Time,sysvector('vehicle_air_data.baro_alt_meter').Data);
```

Comment:
Can you either add here or where the function is called a check if this topic is logged.

Reply:
The reason why this is not working when you test this with the FAT log file is because we forgot to enable the logging of vehicle_air_data through the SD card during that flight. The general question is how we handle this:
a) Just adding a "was this logged" check here does not make sense, so we would either
b) add checks everywhere or
c) nowhere.
I vote for c), because 1) once we have a stable logging configuration "forgetting" topics should ont happen anymore and 2) the matlab warning "XXX is not a key in sysvector" is quite clear, so everybody knows what to do then.

Reply:
I actually have those checks for all variables. Almost all checks are in 'DisplayPX4LogData.m' though.

Usually I do the checks in 'DisplayPX4LogData.m' if this topic is used for the majority of the plots in the function. If only for one subplot as it would be the case here I do the check in the function so that the rest still can be plotted.